README.md

```diff
@@ -38,41 +43,99 @@ on to create a full desktop environment.
 Labwc tries to stay in keeping with [wlroots] and [sway] in terms of general
 approach and coding style.
 
-Labwc only understands [wayland-protocols] &amp; [wlr-protocols], and it cannot
-be controlled with dbus, sway/i3-IPC or other technology. The reason for this is
-that we believe that custom IPCs and protocols create a fragmentation that
-hinders general Wayland adoption.
+Labwc has no reliance on any particular Desktop Environment, Desktop Shell or
+session. Nor does it depend on any UI toolkits such as Qt or GTK.
+
+### 1.1 Why?
+
+We believe that there is a need for a simple Wayland compositor which strikes a
+balance between minimalism and bloat approximately at the level where openbox
+resides in the X11 domain. This does not mean that it should look square and
+pixelated like something out of the 1990s or indeed that the improved
+performance offered by Wayland should be compromised.
+
+It is important to us that scope is tightly controlled so that the compositor
+matures into production quality. On the whole, we value robustness, reliability
+stability and simplicity over new features. This is because we believe that
+coming up with and implementing new ideas is easy, but making them stick and
+work for a long time hard. We want the latter.
+
+[KDE/kwin] and [GNOME/mutter] are great compositors offering brilliant
+experiences, but at ca 200k and 400k lines-of-code respectively and with heavy
+reliance on their respective eco-systems, they cannot be an elegant solution to
+the need for a frugal, independent, Wayland stacking compositor which is the
+space that labwc occupies.
```

Comment:
Non-native english speaker here but to me "occupies" somewhat sounds like we'd be saying that labwc is the only one in that space. Maybe "resides in" would fit better?

I am also not sure if we need to point out the SLOC for kwin and mutter here. Or even need to mention them in the first place. In general I am more of a fan of pointing out our strong points (and weaknesses) than to say why we are better than $xyz. I still like "the boring compositor" btw :p

Reply:
> Non-native english speaker here but to me "occupies" somewhat sounds like we'd be saying that labwc is the only one in that space. Maybe "resides in" or "aims for" would fit better?

Ending a sentence with a [preposition](https://academicguides.waldenu.edu/writingcenter/grammar/prepositions#:~:text=A%20preposition%20is%20a%20word,,%22%20and%20%22to.%22) ("for" or "in" in this case) is poor English.

"occupies" is fine by me. More than one entity can occupy the same space.

Reply:
On further thought maybe something like :

_which is the space that labwc strives to occupy._

Reply:
>  In general I am more of a fan of pointing out our strong points (and weaknesses) than to say why we are better than $xyz.

This is a good point. Maybe we could just leave out the paragraph that mentions Mutter and KWin. Saying "we believe there is a need for" labwc is enough without mentioning other compositors by name.

Reply:
We could also spin this a bit more in explaining our own backgrounds, coming from low resource DEs like Mate / XFCE / Openbox and searching for a simple wayland replacement compositor and then finding and enhancing labwc / creating it in the first place (johanmalm).

I am not sure how to properly formulate this but instead of saying "We believe there is a need for labwc because others are bad" we could explain our personal need for it.

Sometimes I suck at putting my thoughts into words but I hope I got my point across.